GuysNight.LethalCompanyMod.BalancedItems/Plugin.cs

```diff
@@ -10,7 +12,9 @@ public class Plugin : BaseUnityPlugin {
 			SharedComponents.Logger = Logger;
 			Harmony.CreateAndPatchAll(Assembly.GetExecutingAssembly());
 			SharedComponents.Logger.LogInfo($"Plugin {PluginInfo.PLUGIN_GUID} is loaded!");
+			SharedComponents.ConfigFile = new ConfigFile(Path.Combine(Paths.ConfigPath, $"{PluginInfo.PLUGIN_NAME}.cfg"), true);
 
+			SharedComponents.Logger.LogInfo("Default overrides specified in code:");
```

Comment:
Update this message to indicate that config values are still respected. Or maybe remove this entirely since code overrides are in the config too. It's helpful for debugging, but probably not at the cost of confusing consumers.